Packs/Druva/Integrations/Druva/Druva.yml

```diff
@@ -198,8 +259,54 @@ script:
       description: ID of endpoint resource to be decommissioned
     description: 'Wipes remotely an infected Endpoint Resource and deletes a quarantined Snapshot. This command remote wipes data from the endpoint. This action cannot be undone.'
     execution: true
-  dockerimage: demisto/python3:3.9.8.24399
-  isfetch: false
+  - name: druva-find-user
+    arguments:
+    - name: user_string
+      required: true
+      description: Complete user name or p
+    outputs:
+    - contextPath: Druva.User.userID
+      description: The full string or prefix from the user name to search.
+    description: Finds user information for a specific username.
+  - name: druva-find-userDevice
+    arguments:
+    - name: userID
+      required: true
+      description: The userID to search.
+    outputs:
+    - contextPath: Druva.Resource.resourceID
+      description: The ID of the Resource.
+      type: string
+    - contextPath: Druva.Resource.resourceType
+      description: The type of the Resource.
+      type: string
+    description: Finds device information for a specific user.
+  - name: druva-find-sharePointSites
+    arguments:
+    - name: search_string
+      required: true
+      description: The full string or prefix from the share point url to search.
+    outputs:
+    - contextPath: Druva.Resource.resourceID
+      description: The ID of the Resource.
+    - contextPath: Druva.Resource.resourceType
+      description: The type of the Resource.
+    description: 'Find all share point resources with given user name '
+  - name: druva-find-sharedDrives
+    arguments:
+    - name: search_string
+      required: true
+      description: The full string or prefix from the share drive name to search.
+    outputs:
+    - contextPath: Druva.Resource.resourceID
+      description: The ID of the Resource.
+    - contextPath: Druva.Resource.resourceType
+      description: The type of the Resource.
+    description: Finds shared drives resources specific to share drive name
+  dockerimage: demisto/python3:3.10.10.48392
   runonce: false
   subtype: python3
+  longRunningPort: true
```

Comment:
is this a long running integration? if not, lets remove this

Reply:
Removed long running integration.